tests/suites/test_suite_ssl.function

```diff
@@ -1374,7 +1374,9 @@ static int build_transforms( mbedtls_ssl_transform *t_in,
                              md1, maclen,
                              &t_out->psa_mac_enc ) == PSA_SUCCESS );
 
-        psa_set_key_usage_flags( &attributes, PSA_KEY_USAGE_VERIFY_HASH );
+        /* mbedtls_ct_hmac() requires the key to be exportable */
```

Comment:
I need to set:
```c
if( cipher_info->mode == MBEDTLS_MODE_CBC ||
        cipher_info->mode == MBEDTLS_MODE_STREAM )
...
        if( cipher_info->mode == MBEDTLS_MODE_STREAM ||
            etm == MBEDTLS_SSL_ETM_DISABLED )
            psa_set_key_usage_flags( &attributes, PSA_KEY_USAGE_EXPORT |
                                                  PSA_KEY_USAGE_VERIFY_HASH );
```
for all tests to pass, unlike in ssl_tls12_populate_transform() where only:
```c
        if( transform->psa_alg == PSA_ALG_CBC_NO_PADDING &&
             transform->encrypt_then_mac == MBEDTLS_SSL_ETM_DISABLED )
```
is needed.

Is NULL cipher supported only in the test suite ?

Reply:
OK ignore, with compat.sh and NULL removed from EXCLUDE list, I can see some fails.

Reply:
Ah, good, that's what I was going to suggest :)

(Side note: if the code was organized a bit better, we'd only use `cf_hmac()` for CBC without EtM, but the code organization is not optimal, so we're also using it for stream - which currently is equivalent to NULL since we removed RC4.)